runtime/src/serde_snapshot.rs

```diff
@@ -284,7 +302,8 @@ pub(crate) fn bank_from_streams<R>(
     serde_style: SerdeStyle,
     snapshot_streams: &mut SnapshotStreams<R>,
     account_paths: &[PathBuf],
-    unpacked_append_vec_map: UnpackedAppendVecMap,
+    storage: DashMap<Slot, SlotStores>,
+    next_append_vec_id: AtomicU32,
```

Comment:
and I wonder if this tuple of storage and next_append_vec_id should be a struct or type?